src/decode-tcp.h

```diff
@@ -158,22 +165,20 @@ typedef struct TCPVars_
     bool md5_option_present;
     bool ao_option_present;
     bool ts_set;
+    bool sack_ok;
+    bool mss_set;
+    bool tfo_set;
+    uint8_t wscale_set : 1;
```

Comment:
why not a bool ?

Reply:
I've made all of them bitfields for max compactness